src/MOI_wrapper.jl

```diff
@@ -17,7 +17,7 @@ mutable struct Optimizer <: MOI.AbstractOptimizer
     variable_index::Vector{Int} # MOI.VariableIndex -> state/input/mode index
     lower::Vector{Float64}
     upper::Vector{Float64}
-    start::Vector{Float64}
+    start::Vector{Union{Float64, MOI.Interval{Float64}}}
```

Comment:
```suggestion
    start::Vector{MOI.Interval{Float64}}
```

Reply:
We should modify `set` for `VariablePrimalStart` and create an interval with two equal bounds there

Reply:
for the case where start is only a value, right?